enclave/Enclave.edl

```diff
@@ -26,6 +26,7 @@ enclave {
 	from "sgx_time.edl" import *;
 	from "sgx_env.edl" import *;
 	from "sgx_thread.edl" import *;
+	from "sgx_pipe.edl" import *;
```

Comment:
I wonder about the pipe, what feature does it bring?

Reply:
It's a library/symbols required to run `mio` inside the enclave. We could however remove it again for the moment, since my current implementation doesn't use `mio` to run the web-socket server (I had past versions where that was the case).